test/fail_compilation/ice12673.d

```diff
@@ -1,3 +1,13 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/ice12673.d(13): Error: static assert:  `__traits(compiles, ()
+{
+__error__
+}
```

Comment:
@wilzbach - Are you sure it isn't the d_do_test program that's messing up the newlines on Windows?

Reply:
Not sure, the Windows support is rather hackish, but from what I can tell it only calls unifyNewLines for the expected output.

Reply:
The output file should be opened with mode "wb" (https://github.com/dlang/dmd/blob/master/test/tools/d_do_test.d#L706), otherwise multiple `\r` are written (https://github.com/dlang/dmd/blob/master/test/tools/d_do_test.d#L401).

Reply:
Ugh, let's try that then.

Reply:
Hmm now another test fails:

```
Test fail_compilation\ice11982.d failed: 

expected:

----

fail_compilation\ice11982.d(16): Error: basic type expected, not `scope`

fail_compilation\ice11982.d(16): Error: found `scope` when expecting `;` following statement

fail_compilation\ice11982.d(16): Error: basic type expected, not `}`

fail_compilation\ice11982.d(16): Error: missing `{ ... }` for function literal

fail_compilation\ice11982.d(16): Error: C style cast illegal, use `cast(funk)function _error_()

{

}

`

fail_compilation\ice11982.d(16): Error: found `}` when expecting `;` following statement

fail_compilation\ice11982.d(17): Error: found `End of File` when expecting `}` following compound statement

----

actual:

----

fail_compilation\ice11982.d(16): Error: basic type expected, not `scope`

fail_compilation\ice11982.d(16): Error: found `scope` when expecting `;` following statement

fail_compilation\ice11982.d(16): Error: basic type expected, not `}`

fail_compilation\ice11982.d(16): Error: missing `{ ... }` for function literal

fail_compilation\ice11982.d(16): Error: C style cast illegal, use `cast(funk)function _error_()



{



}



`

fail_compilation\ice11982.d(16): Error: found `}` when expecting `;` following statement

fail_compilation\ice11982.d(17): Error: found `End of File` when expecting `}` following compound statement

----
```

Reply:
> Hmm now another test fails:

Indeed, it's hard to know ahead of time what might fail the comparison test, but it's running into the dozens.

I'd rather attempt to fix the testsuite rather than remove any more.